web/src/compositions/useTheme.ts

```diff
@@ -1,23 +1,11 @@
-import { computed, ref, watch } from 'vue';
+import { BasicColorSchema, useColorMode } from '@vueuse/core';
+import { computed, watch } from 'vue';
 
-export enum Theme {
-  Auto = 'auto',
-  Light = 'light',
-  Dark = 'dark',
-}
-
-const LS_THEME = 'woodpecker:theme';
-const activeTheme = ref(Theme.Auto);
-
-function resolveAuto(theme: Theme) {
-  if (theme === Theme.Auto) {
-    return window.matchMedia('(prefers-color-scheme: dark)').matches ? Theme.Dark : Theme.Light;
-  }
-  return theme;
-}
+const { system, store } = useColorMode();
+const resolvedTheme = computed(() => (store.value === 'auto' ? system.value : store.value));
```

Comment:
Seems not to be needed from the code (docs seem to be misleading a bit): https://github.com/vueuse/vueuse/blob/main/packages/core/useColorMode/index.ts#L142